src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  let discount = 0;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    discount = -20;
```

Comment:
i think - we don't need `let` variable. We can return it immediately in the `if` condition
```suggestion
    return totalRent - SHORT_TERM-DISCOUNT;
```